auth-mailchimp-sync/extension.yaml

```diff
@@ -67,10 +67,11 @@ resources:
 params:
   - param: LOCATION
     type: select
-    label: Deployment location
+    label: Cloud Functions location
     description: >-
-      Where should the extension be deployed? For help selecting a location,
-      refer to the [location selection
+      Where do you want to deploy the functions created for this extension? 
+      You usually want a location close to your database. For help selecting a 
```

Comment:
there's no database involved in this extension. It actually doesn't really matter where the cloud function is created for  this extension.

Reply:
removed